safe_transaction_service/history/serializers.py

```diff
@@ -119,8 +122,11 @@ def validate_signature(self, signature: bytes):
         )
 
         safe_owners = get_safe_owners(safe_address)
+        safe_hash_preimage = select_safe_encoded_message_hash_by_safe_version(
```

Comment:
Should we move this to the safe-eth-py library? I mean to return the right safe_tx_hash_preimage directly from the safe_tx taking account the version of the Safe, without have this util in the tx-service. 
What do you think? @Uxio0 ?

Reply:
I think that we don'r need this function `select_safe_encoded_message_hash_by_safe_version`